crates/api_models/src/payments.rs

```diff
@@ -3687,6 +3702,12 @@ pub struct KlarnaSdkPaymentMethodResponse {
     pub payment_type: Option<String>,
 }
 
+#[derive(Debug, Clone, Eq, PartialEq, serde::Deserialize, serde::Serialize, ToSchema)]
+
+pub struct KlarnaCheckoutPaymentMethodResponse {
```

Comment:
what details from the response are we storing here? if not needed, please remove